### ISSUE TITLE
Solution not working

### ISSUE THREAD

**Issue by graphefruit** [manfredsteyer/angular-crud#10]
Hello @manfredsteyer,
I cloned the solution, and tried your sample.
```
Your global Angular CLI version (6.2.2) is greater than your local
version (1.5.5). The local Angular CLI version is used.

To disable this warning use "ng config -g cli.warnings.versionMismatch false".
Error: Could not find an NgModule for the new component. Use the skip-import option to skip importing components in NgModule.
Could not find an NgModule for the new component. Use the skip-import option to skip importing components in NgModule.

```
Any ideas?
Even with `--skip-import` its not working

I also tried multiple samles, but all are not working, were there breaking changes?
Thanks for any advanced help.
Graphefruit

**Comment by hungcao181** [comment]
Make sure to follow the instruction here (it is a little bit confusing to me too):

> 4. In the same folder, run the following Angular CLI based command:
> ```
> ng g angular-crud:crud-module hotel" 
> ```
> 

So to run that command, you have to cd to the hotel folder which is created at step 3.

**Comment by mbirabhadra** [comment]
Make sure in the app.module.ts the path in the import is correct. It was wrong by default for me. I corrected it to
`import { HotelModule } from './hotel/hotel.module';`

**Comment by mbirabhadra** [comment]
In the instruction it says:
Also, open the file sidebar.component.ts and add a link to the generated route:

But It should be `sidebar.component.html` instead of the ts file.

**Comment by davidmpaz** [comment]
Hi,

i am facing the same issue. I downloaded the project. Installed dependencies. I go inside `src/app/hotel` with the `model.json` file inside and try to run the command: 
```
npm run ng g angular-crud:crud-module hotel -- --skip-import
```
but no success. Same error reported here. When i try to execute it with `--skip-import` i get then this error:
```
Model file model.json does not exist.
```

I have also tried to pass all parameters manually with absolute paths but no success. Any idea ?

thanks in advance,
David

UPDATE:
Just to note that i am running `ng` tool from inside the repository, which makes sense to me since i will use the cli tool installed for the project specifically and not a global one that can be out of sync with project versions. 

When running the generator from a global installation of `ng` it works like a charm. Maybe some polishing in regards path scopes is needed?
